### PR TITLE
lesson 1_3

### PR DESCRIPTION
**Pull request by verafes**
use States
addToDo, TodoListItems handling
props

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
-import TodoList from './TodoList';
+import TodoList, { initialTodoList } from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
 function App() {
+    const [newTodoList, setTodoList] = useState( initialTodoList );
+    let len = newTodoList.length;
+
+    const addTodo = (title) => {
+        const newId = newTodoList[len - 1].id + 1;
```
Comment (AnnaPestova1) [inline src/App.jsx:11]: You did a great job to add multiple todos. Great!

File: src/AddTodoForm.jsx [hunk 2]
```diff
@@ -1,8 +1,15 @@
-function AddTodoForm() {
+function AddTodoForm(todoprops) {
```
Comment (AnnaPestova1) [inline src/AddTodoForm.jsx:1]: for consistency React developers using word 'props' or refactor props (we'll learn it soon).

File: src/App.jsx [hunk 3]
```diff
@@ -1,15 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css'
-import TodoList from './TodoList';
+import TodoList, { initialTodoList } from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
 function App() {
+    const [newTodoList, setTodoList] = useState( initialTodoList );
```
Comment (AnnaPestova1) [inline src/App.jsx:7]: when we are using useState by convention we use the name for the function state setter in useState with the word 'new' in the beginning and state name. [todo, newTodo], [word, newWord] etc.

**Review by AnnaPestova1 — APPROVED** [review]
Vera! Great job! You did an extra and it looks great! You can merge your PR.

File: src/AddTodoForm.jsx (hunk 2 above)
Comment (verafes) [reply src/AddTodoForm.jsx:1]: Understood, thanks.

File: src/App.jsx (hunk 3 above)
Comment (verafes) [reply src/App.jsx:7]: Got it, thank you!

**Pull request merged**